### ISSUE TITLE
TypeError: forward() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by parkhy0106** [ZhendongWang6/Uformer#14]
test_in_any_resolution.py has an error and It is the full stack of error message.

```
 `Traceback (most recent call last):
  File "/content//Uformer/test_in_any_resolution.py", line 104, in <module>
    rgb_restored = model_restoration(rgb_noisy, 1 - mask)
  
  File "/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  
  File "/usr/local/lib/python3.7/dist-packages/torch/nn/parallel/data_parallel.py", line 159, in forward
    return self.module(*inputs[0], **kwargs[0])
  
  File "/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)

TypeError: forward() takes 2 positional arguments but 3 were given`
```

Actually, I was trying with my custom data but Its shape is the same as sidd data so I can't understand why.

**Comment by vinthony** [comment]
hi, you also need to update the model.py to apply this script

**Comment by parkhy0106** [comment]
Yeah, It works! and Don't know why but there is a white rim in the predicted result.

**Comment by parkhy0106** [comment]
![0651e930-62f0-4852-b456-a87bd84a0cfd](https://user-images.githubusercontent.com/53888831/127759045-b910ab97-7352-423b-b2a0-917b52350ac8.png)

**Comment by vinthony** [comment]
Hi, thanks for the feedback. 

Have you masked out the invalid pixels for validation?

**Comment by parkhy0106** [comment]
Nope. I inferred with not masked out files and not edited test_in_any_resolution.py

**Comment by vinthony** [comment]
Hi, I have tried this code by myself. However, it seems fine in my case.

As shown in the above image, from left to right is the predicted image, gt image, input image, and corresponding mask. There is no white rim around.

![image](https://user-images.githubusercontent.com/4397546/127763848-4c32a139-b0c9-4bf4-a04e-2179bd6aca73.png)


By the way, I also visualize the predicted image before the mask, it also looks fine.
![0000-0000](https://user-images.githubusercontent.com/4397546/127763973-fec90321-fb5d-4f77-bf46-9c2d77e8a8f4.png)

So it is kind of weird in this case. Have you tried to modify the model?

**Comment by parkhy0106** [comment]
Nope. I didn't modify the models or any kind of arch Uformer has. Instead, I suspect that 

1. My training epoch is too less. 
_-(I early stopped because I don't have that much time and computing power and also the train soon overfitted because I was testing Lightnoise denoising without GAN)-_
But this is weird because normal test.py doesn't occur this problem

2. My test data is too high resolution
 _And I tried to fix this by splitting my photo into many pieces and It works little but didn't fix this problem perfectly._ 

3. My train-valid data is not fine so makes bias.
 _There is no chance to check this problem. This data isn't mine so I can't check this._


For now, I just decided to fix this by using test.py. What could be the causation?
